installation-usage.md

```diff
@@ -6,6 +6,8 @@ The feature scope of the seccomp-operator is right now limited to:
 
 - Enabling `ConfigMap`s to store seccomp profiles.
 - Synchronizing seccomp profiles across all worker nodes.
+- Validate if a node supports seccomp and do not synchronize if not.
+- Validate if a profile is syntactical correct and do not synchronize if not.
```

Comment:
nit: we could keep all sentences on the same tense, by removing the present continuous from the first two:

```suggestion
- Enable `ConfigMap`s to store seccomp profiles.
- Synchronize seccomp profiles across all worker nodes.
- Validate if a node supports seccomp and do not synchronize if not.
- Validate if a profile is syntactically correct and do not synchronize if not.
```

Reply:
Adapted as suggested :+1: